.github/workflows/automerge_scan.yml

```diff
@@ -50,6 +53,9 @@ jobs:
             else
               echo "$url Check failed!!!"
               echo $check | jq
+              if [[ $created_at > $(date --date "1 day ago" -u +"%FT%TZ") ]];then
```

Comment:
Only trigger a rebuild when Azure.sonic-buildimage failed, and if retry has added, we should not retry every day.

Reply:
We can't get attempt number.
It is not for every day. It is for one day.